data-visualize.qmd

```diff
@@ -778,7 +774,7 @@ You will learn about many other geoms for visualizing distributions of variables
 
 ### Exercises
 
-1.  Which variables in `mpg` are categorical?
+1.  Which variables in `mpg` are categorical in the context of the data?
```

Comment:
In what other context would you consider them?

Reply:
I agree, I don't think this is an improvement. It was an attempt to distinguish between R thinking a 0/1 variable could be numerical when it's actually categorical. But we haven't introduced data types yet here so I'm going to revert this, I think it complicates instead of clarifying.